main/javascript-modifications/ses-reference.md

```diff
@@ -0,0 +1,286 @@
+object should only be handled by very trustworthy code.
+
+```js
+lockdown(); // consoleTaming defaults to 'safe'
+// or
+lockdown({ consoleTaming: 'safe' }); // Wrap start console to show deep stacks
+// vs
+lockdown({ consoleTaming: 'unsafe' }); // Leave original start console in place
+```
+
+## `errorTaming` Options
+
+The `errorTaming` default `'safe'` setting makes the stack trace inaccessible
+from error instances alone. It does this on v8 engines (Chrome, Brave, Node). 
+Note that it is **not** hidden on other engines, leaving an information
+leak available. It reveals information only as a powerless string. 
+
+In JavaScript the stack is only available via `err.stack`, so some 
+development tools assume it is there. When the information leak is tolerable, 
+the `'unsafe'` setting preserves `err.stack`'s filtered stack information.
+
+`errorTaming` does not affect the `Error` constructor's safety. 
+After calling `lockdown`, the tamed `Error` constructor in the
+start compartment follows ocap rules. Under v8 it emulates most of the
+magic powers of the v8 `Error` constructor&mdash;those consistent with the
+discourse level of the proposed `getStack`. In all cases, the `Error`
+constructor shared by all other compartments is both safe and powerless.
+```js
+lockdown(); // errorTaming defaults to 'safe'
+// or
+lockdown({ errorTaming: 'safe' }); // Deny unprivileged access to stacks, if possible
+// vs
+lockdown({ errorTaming: 'unsafe' }); // Stacks also available by errorInstance.stack
+```
+
+## `stackFiltering` Options
+
+`stackFiltering` trades off stronger stack traceback filtering to
+minimize distractions vs completeness for tracking down bugs in
+obscure places. 
+
+The default `'concise'` setting removes "noise" from the full distributed
+stack traces, in particularly artifacts from low level infrastructure. It
+only works on v8 engines.
+
+With the `'verbose'` setting, the `console` displays the full raw stack
+information for each level of the "deep stack", tracing back through the
+[eventually sent messages](https://github.com/tc39/proposal-eventual-send)
+from other turns of the event loop. This makes JavaScript's already voluminous 
+error stacks even more so. However, this is sometimes useful for finding bugs
+in low level infrastructure.
+
+Both settings are safe. Stack information will
+or will not be available from error objects according to the `errorTaming`
+option and the platform error behavior.
+
+```js
+lockdown(); // stackFiltering defaults to 'concise'
+// or
+lockdown({ stackFiltering: 'concise' }); // Preserve important deep stack info
+// vs
+lockdown({ stackFiltering: 'verbose' }); // Console shows full deep stacks
+```
+
+## `overrideTaming` Options
+
+**tyg todo: Honestly not fond of this. If you can do a better job of giving a very brief
+summary of what the settings just do without having to explain a lot of background, please
+do.**
```

Comment:
```suggestion
We recommend explicitly selecting `overrideTaming` of `min` when beginning a project because debugging will be more pleasant. You may need to relax it back to the default of `moderate` third-party shimming code interferes with `lockdown`. By way of background, these are two treatments to address a language design mistake that can interfere with the SES shim.
```

I defer to @erights to verify the correctness of this summary or propose a better alternative.